shared/test/test_tables.rb

```diff
@@ -0,0 +1,67 @@
+require 'minitest/autorun'
+require 'rack/test'
+require File.expand_path '../../../deployment', __FILE__
+require File.expand_path '../../middleware/channels_api', __FILE__
+require File.expand_path '../../middleware/tables_api', __FILE__
+
+ENV['RACK_ENV'] = 'test'
+
+class TablesTest < Minitest::Unit::TestCase
+
+  def create_channel
```

Comment:
This could be more trouble than it's worth, as the number of duplicated lines is small, and channels test both goes into more depth and is configured differently. This is good to keep in mind though if more common setup code emerges.